src/components/CardBoard.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+    <div class="board">
+        <add-todo @fetchTodo='fetchTodo'></add-todo>
+        <div class="card-board">
+                <card-container  @fetchTodo='fetchTodo' v-for="(category, index) in categories" :key="index" :category="category" :showTasks='showTasks'>
+                    
```

Comment:
spacing berlebihan

Reply:
kalo attributes element kepanjangan tolong di enter 
misal:
<card container
  @fetchTodo=''
  v-for=''
>

Reply:
ok mas makasih feedbacknya